index.html

```diff
@@ -7,50 +7,48 @@
     <title>Commonplace Cooperative</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  
-  <header>
-    <!-- Header section -->
-    <div class="header">
-      <!-- Nav Bar -->
-        <div class="nav-bar">
-           <div class="left-side"> <!--rename elements to <nav> ?-->
-            <img id="logo" src="/images/logo.png" alt="commonplace-logo">
-            <p id="commonplace-coop">commonplace.coop</p>
+  <body>
+    <header>
+      <!-- Header section -->
+      <div class="header">
+        <!-- Nav Bar -->
+          <div class="nav-bar">
+             <div class="left-side"> <!--rename elements to <nav> ?-->
+              <img id="logo" src="/images/logo.png" alt="commonplace-logo">
+              <p id="commonplace-coop">commonplace.coop</p>
+            </div>
+            <div class="right-side">
+              <nav class="site-routes">
+                <a href="#" id="services">services</a>
+                <a href="#" id="media">media</a>
+                <a href="#" id="events">events</a>
+                <a href="#" id="about">about</a>
+                <a href="#" id="join">join</a>
+              </nav>
+            </div>
           </div>
-          <div class="right-side">
-            <nav class="site-routes">
-              <a href="#" id="services">services</a>
-              <a href="#" id="media">media</a>
-              <a href="#" id="events">events</a>
-              <a href="#" id="about">about</a>
-              <a href="#" id="join">join</a>
-            </nav>
+        </div>
+      <!-- section 1 -->
+        <div class="white-space-one">
+          <div class="tagline-one">
+            <h3 id="commonplace">COMMONPLACE</h3>
+              <div class="text-right">
+                <p id="where">where</p> <!-- -->
+                <p id="cooperation">cooperation</p>
```

Comment:
There's a lot of styling by ID. We generally only style by ID if there's something very special about that element, like a Floating Action Button (a UI concept from Material, where there is one large button on the screen to perform a major action).